Server/models/Check.js

```diff
@@ -31,4 +31,27 @@ const CheckSchema = mongoose.Schema(
   }
 );
 
+/**
+ * When a check is created, we should update the stauts of the associated monitor
```

Comment:
I'm trying to learn more and more about documentation these days and try to take advantage of AI and Chat-GPT tools for such purposes. So, here's what I got as another suggestion of comments and documentation for this code snippet:

```
/**
 * Middleware function executed before saving a check.
 * Updates the status of the associated monitor if it has changed.
 *
 * @param {function} next - Callback to proceed to the next middleware.
 */
```

What do you think?
Can we replace it with this one?

Reply:
AI can be a helpful tool, but I prefer to write my own documentation.  For example in this case it has missed out on the fact that the default value of the monitor status is true, which is useful information for anyone working on this method.

The js DOC is good though, we can include that :+1:

Reply:
Yeah. We can combine them and improve it 👍🏻